packages/block-library/src/buttons/edit.js

```diff
@@ -20,16 +23,42 @@ const alignmentHooksSetting = {
 	isEmbedButton: true,
 };
 
-function ButtonsEdit() {
+function ButtonsEdit( { attributes, setAttributes } ) {
+	const { blockDirection } = attributes;
+
 	return (
-		<Block.div>
+		<Block.div
+			className={ `${
+				blockDirection === 'vertical' ? 'is-direction-vertical' : ''
```

Comment:
Generally the codebase uses `classnames` for this sort of logic:
```
className={ classnames( { 'is-direction-vertical': blockDirection === 'vertical } ) }

Reply:
Lets also shorten the classname to `is-vertical` to match the one used in the navigation block for this purpose.

Reply:
Done, thank you! ❤️